src/elements/content-sidebar/activity-feed/task-new/TaskStatus.js

```diff
@@ -13,27 +12,39 @@ import messages from './messages';
 import type { TaskStatus } from '../../../../common/types/tasks';
 
 import './TaskStatus.scss';
+// $FlowFixMe
+import LabelPill from '../../../../components/label-pill';
 
 type Props = {|
     status: TaskStatus,
 |};
 
 const statusMessageKeyMap = {
-    [TASK_NEW_APPROVED]: messages.tasksFeedApprovedLabel,
-    [TASK_NEW_COMPLETED]: messages.tasksFeedCompletedLabel,
-    [TASK_NEW_REJECTED]: messages.tasksFeedRejectedLabel,
-    [TASK_NEW_NOT_STARTED]: messages.tasksFeedInProgressLabel,
-    [TASK_NEW_IN_PROGRESS]: messages.tasksFeedInProgressLabel,
+    [TASK_NEW_APPROVED]: messages.taskFeedApprovedUppercaseLabel,
+    [TASK_NEW_COMPLETED]: messages.taskFeedCompletedUppercaseLabel,
+    [TASK_NEW_REJECTED]: messages.taskFeedRejectedUppercaseLabel,
+    [TASK_NEW_NOT_STARTED]: messages.taskFeedInProgressUppercaseLabel,
+    [TASK_NEW_IN_PROGRESS]: messages.taskFeedInProgressUppercaseLabel,
+};
+
+const typeKeyMap = {
+    [TASK_NEW_APPROVED]: 'success',
+    [TASK_NEW_COMPLETED]: 'success',
+    [TASK_NEW_REJECTED]: 'error',
+    [TASK_NEW_NOT_STARTED]: 'default',
+    [TASK_NEW_IN_PROGRESS]: 'default',
 };
 
 const Status = React.memo<Props>(({ status }: Props) => (
     <FormattedMessage
         {...messages.tasksFeedStatusLabel}
         values={{
             taskStatus: (
-                <span className={`bcs-TaskStatus-message ${camelCase(status)}`}>
-                    <FormattedMessage {...statusMessageKeyMap[status]} />
-                </span>
+                <LabelPill.Pill type={typeKeyMap[status]}>
+                    <span>
```

Comment:
Why is this extra element needed?

Reply:
Removing.

Reply:
use `LabelPill.Text` on pill text contents